helm/postgres-openidm/values.yaml

```diff
@@ -23,17 +19,19 @@ imageTag: "9.5.4"
 # For postgres we  use the standard vanilla image on the Docker hub.
 postgres:
   image: postgres
-  tag: 9-alpine
+  tag: 10.5-alpine
 
 postgresUser: openidm
 postgresPassword: openidm
 
-## Persist data to a persitent volume.
+## Persist data
 persistence:
   enabled: true
-  storageClass: generic
   accessMode: ReadWriteOnce
-  size: 8Gi
+
+storageClassName: standard
```

Comment:
Is this supposed to be at top level?

Reply:
yes -I  moved to top level. There really isn't a good reason to nest this

Reply:
Ok but there is the persistence section and this would be a logic fit under it as before?

Reply:
I will top level those as well...

Reply:
done